src/scenes/home/jobs/featuredJobs.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "title": "Software Engineer",
+    "source": "Microsoft",
+    "sourceUrl": "https://www.microsoft.com",
+    "city": "Redmond",
+    "state": "Washington",
+    "country": "United States",
+    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
+    "status": "active",
+    "remote": false,
+    "tags": [
+      "C#",
+      "ASP.Net",
+      "DevOps"
+    ]
+  },
+  {
+    "title": "QA Engineer",
+    "source": "Apple",
+    "sourceUrl": "https://www.apple.com",
+    "city": "San Francisco",
+    "state": "California",
+    "country": "United States",
+    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. ",
+    "status": "active",
+    "remote": true,
+    "tags": [
+      "C#",
+      "ASP.Net",
+      "DevOps"
+    ]
+  },
+  {
+    "title": "DevOps Engineeer",
+    "source": "GitHub",
+    "sourceUrl": "https://www.github.com",
+    "city": "Austin",
+    "state": "texas",
```

Comment:
Texas

Reply:
This JSON file is just mock data for now. Its content will be replaced with actual featured jobs, or the JSON file will discarded completely once the API endpoint is ready.